Header header

# automatic or triggered mode
bool automatic

# in triggered mode, each new event is marked by a change in the counter
uint8 event_counter
